### PR TITLE
add checks for custom tokens

### PR DESCRIPTION
**Pull request by anna-carroll**
## Motivation

Custom tokens weren't covered in our deploy checks

## Solution

Add some checks :) 

## PR Checklist

- [ ] Added Tests
- [ ] Updated Documentation
- [ ] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

File: packages/new-deploy/src/bridge/BridgeContracts.ts [hunk 1]
```diff
@@ -623,6 +623,59 @@ export default class BridgeContracts extends AbstractBridgeDeploy<config.EvmBrid
...
+        expect(decimals).to.eq(custom.decimals);
+
+        const tokenId =
+          await this.tokenRegistryContract.representationToCanonical(
+            addresses.proxy,
+          );
+        expect(tokenId.domain).to.eq(custom.token.domain);
+        expect(utils.equalIds(tokenId.id, custom.token.id)).to.be.true;
+
+        // TODO: calculate tokenId in javascript
```
Comment (prestwich) [inline packages/new-deploy/src/bridge/BridgeContracts.ts:673]: is this a new TODO issue, or a TODO on this PR?

Comment (anna-carroll) [reply packages/new-deploy/src/bridge/BridgeContracts.ts:673]: it's a TODO on this PR.. i was looking for utils for this in the SDK but couldn't find

**Review by kekonen — APPROVED** [review]
lgtm

**Comment by anna-carroll** [comment]
added in to this PR: https://github.com/nomad-xyz/monorepo/pull/310/files

**Pull request closed without merging**